python/setup.py

```diff
@@ -77,7 +77,7 @@
 extras = {
     "debug": [],
     "dashboard": ["requests"],
-    "serve": ["uvicorn", "pygments", "werkzeug", "flask", "pandas", "blist"],
+    "serve": ["uvicorn", "flask", "blist", "prometheus_client"],
```

Comment:
This should probably be an optional dependency only for those who are exporting to prometheus